### PR TITLE
Google calendar

### PR DESCRIPTION
**Pull request by olsona**
Adding tests

### CODE REVIEW

File: portability-core/src/main/java/org/dataportabilityproject/serviceProviders/google/GoogleStaticObjects.java [hunk 1]
```diff
@@ -64,4 +63,9 @@ public static HttpTransport getHttpTransport() {
   public static FileDataStoreFactory getDataStoreFactory() {
     return DATA_STORE_FACTORY;
   }
+
+  /**
+   * Static values used in converters
```
Comment (holachuy) [inline portability-core/src/main/java/org/dataportabilityproject/serviceProviders/google/GoogleStaticObjects.java]: Maybe add a little more doc on why MAX_ATTENDEES is set at this value

File: portability-core/src/test/java/org/dataportabilityproject/serviceProviders/google/calendar/GoogleCalendarServiceTest.java [hunk 2]
```diff
@@ -21,72 +21,108 @@
...
+   * Sets up a response for a single event
+   */
+  private void setUpSingleEventResponse() throws IOException {
+    eventListResponse = new Events().setItems(Collections.singletonList(EVENT));
+    when(eventListRequest.execute()).thenReturn(eventListResponse);
   }
 
   @Test
-  public void testExportFirstSet() throws IOException {
+  public void testExportCalendarFirstSet() throws IOException {
```
Comment (holachuy) [inline portability-core/src/test/java/org/dataportabilityproject/serviceProviders/google/calendar/GoogleCalendarServiceTest.java]: Would First or Single be more accurate for this test?

Comment (olsona) [reply portability-core/src/test/java/org/dataportabilityproject/serviceProviders/google/calendar/GoogleCalendarServiceTest.java]: I think the fact that it's testing the export of the first calendar page is more important.

File: portability-core/src/main/java/org/dataportabilityproject/serviceProviders/google/GoogleStaticObjects.java (hunk 1 above)
Comment (olsona) [reply portability-core/src/main/java/org/dataportabilityproject/serviceProviders/google/GoogleStaticObjects.java]: Adding a TODO to evaluate this.

**Pull request merged**